platforms/jvm/toolchains-jvm/src/main/java/org/gradle/jvm/toolchain/JvmVendorSpec.java

```diff
@@ -83,6 +83,14 @@ public abstract class JvmVendorSpec {
 
     public static final JvmVendorSpec SAP = matching(KnownJvmVendor.SAP);
 
+    /**
+     * A constant for using <a href="https://github.com/Tencent/TencentKona-8">Tencent Kona JDK</a> as the JVM vendor.
+     *
+     * @since 8.5
```

Comment:
The version of Gradle on master has changed. You need to update this to `8.6`

Reply:
Thanks very much!
I just pushed a new commit.